pandas/core/arrays/interval.py

```diff
@@ -781,7 +790,9 @@ def shift(self, periods: int = 1, fill_value: object = None) -> "IntervalArray":
 
         empty_len = min(abs(periods), len(self))
         if isna(fill_value):
-            fill_value = self.left._na_value
+            from pandas import Index
+
+            fill_value = Index(self._left)._na_value
```

Comment:
why can't we get the fill value rather than doing this? if we need to do this add copy=False